File: WebSolution/Layered/TFLOBL/Entities/ProjectMgmt/ProjectMembership.cs
namespace Transflower.TFLPortal.TFLOBL.Entities;

public class ProjectMembership
{
    public int Id { get; set; }
    public int ProjectId { get; set; }
    public int EmployeeId { get; set; }
    public string? ProjectRole {get; set;}
    public DateTime ProjectAssignDate { get; set; }
    public DateTime ProjectReleaseDate { get; set; }
    public string? CurrentProjectWorkingStatus { get; set; }
}

File: Solution/UISolution/TFLUIPortal/src/app/shared/draft/projects/Models/projectmembership.ts
export class ProjectMembership{
    constructor(
      public Id: number,  
      public employeeId: number,
      public projectId: number,
      public projectRole: string,
      public projectAssignDate: string,
      // public releaseDate: string,
      public currentProjectWorkingStatus:string
    ){}
  }